### PR TITLE
Transitions popup (for editing order)

### PR DESCRIPTION
**Pull request by ahmedhamodi**
Video showing the feature:
https://www.loom.com/share/3770fa6539264786a1012055677377ca

Reworked the popup styling, see new image below:
image

### CODE REVIEW

File: src/Components/Admin/EnvironmentEditor/environment.js [hunk 1]
```diff
@@ -214,10 +216,13 @@ export default function EnvironmentEditor({
...
-    const onTransitionModalSubmit = async () => {
+    const onTransitionModalSubmit = async (transitions) => {
         setEditTransitionModalOpen(false);
 
-        // TODO: edit transition data upon edit
+        const envData = environment;
+        envData.transitions.data = transitions;
+        const response = await editScenario(envData, handleError);
+        setEnvironment(response.data);
     };
```
Comment (aaronabraham311) [inline src/Components/Admin/EnvironmentEditor/environment.js:226]: Are we firing a request each time we change the transition order?

File: src/Components/Admin/EnvironmentEditor/transitionModal.js [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+    return (
+        <Dialog open={modalOpen} onClose={handleModalClose}>
+            <DialogTitle>Edit Transitions Order</DialogTitle>
+            <DialogContent>
+                {transitions.map((transition, index) => {
+                    return (
+                        <div key={transition.id}>
+                            <h4>{transition.text}</h4>
+                            <h5>
+                                Transition {index + 1} of {transitions.length}
```
Comment (aaronabraham311) [inline src/Components/Admin/EnvironmentEditor/transitionModal.js:64]: Could we reverse the order

**Review by aaronabraham311 — COMMENTED** [review]
Nice work! Couple of comments

File: src/Components/Admin/EnvironmentEditor/environment.js (hunk 1 above)
Comment (ahmedhamodi) [reply src/Components/Admin/EnvironmentEditor/environment.js:226]: When you "submit" the modal, then yes. I can add a quick check for if it's the same order?

File: src/Components/Admin/EnvironmentEditor/transitionModal.js (hunk 2 above)
Comment (ahmedhamodi) [reply src/Components/Admin/EnvironmentEditor/transitionModal.js:64]: What do you mean? Want transition 1 on bottom? I feel like the current order makes more sense

Comment (aaronabraham311) [reply src/Components/Admin/EnvironmentEditor/transitionModal.js:64]: No I mean Transition {index + 1} of {transitions.length} above the actual text

Comment (ahmedhamodi) [reply src/Components/Admin/EnvironmentEditor/transitionModal.js:64]: Oh I see. will do

**Review by aaronabraham311 — APPROVED** [review]

**Pull request merged**